.github/workflows/format_json.yaml

```diff
@@ -1,5 +1,5 @@
 name: 'PBIX Deserializer'
-description: 'Deserializes PBIX Files into json files for easier version control'
+#description: 'Deserializes PBIX Files into json files for easier version control'
```

Comment:
Workflows don't have a description field. Go ahead and delete this line and we can look at deploying it to all of our repos.